tests/test_project_report.py

```diff
@@ -361,30 +391,76 @@ def test_get_sample_info(self, mocked_project_size):
             'adapter2': 'AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT',
             'bwa_version': 1.2,
             'biobambam_sortmapdup_version': 2,
+            'bcl2fastq_version': 'v2.17.1.14',
```

Comment:
It seems like all these packages have hardcoded version numbers: aren't we moving away from this?

Reply:
Nope, all the other tool versions are being read from programs.txt - see line 278.

Reply:
This test checks that the files generated in the `setUp` were parsed accurately